### PR TITLE
Spring pr

### PR DESCRIPTION
**Pull request by jvitrifork**
This reflects the code changes discussed in https://github.com/hapifhir/hapi-fhir-jpaserver-starter/issues/164

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -5,32 +5,31 @@ spring:
...
 
   batch:
     job:
       enabled: false
 
 hapi:
   fhir:
-    defer_indexing_for_codesystems_of_size: 101
+    ### This is the FHIR version. Choose between, DSTU2, DSTU3, R4 or R5
+    fhir_version: DSTU3
```
Comment (jamesagnew) [inline src/main/resources/application.yaml:31]: R4 should be default, as it's the current release

**Review by jamesagnew — COMMENTED** [review]
Looks like this still has CI issues - Overall, cool approach though!

**Comment by jkiddo** [comment]
whoooops ... hadn't seen that. Standby for fix

File: src/main/resources/application.yaml (hunk 1 above)
Comment (jvitrifork) [reply src/main/resources/application.yaml]: now it is

**Comment by jvitrifork** [comment]
@jamesagnew I just checked the documentation. It currently does not mention how to use the fhir_version/'spring boot active' profiles - which means that there actually isn't any documentation to change in regards to this PR - which again means that you can freely accept this PR without having 'forgotten' to update any doc as it isn't present anyway ;)

**Comment by jamesagnew** [comment]
Nice, thanks for the PR!

**Pull request merged**